ios/Plugin/Plugin.swift

```diff
@@ -192,6 +192,21 @@ public class CapacitorGoogleMaps: CustomMapViewEvents {
             }
         }
     }
+    
+    
+    @objc func removeMap(_ call: CAPPluginCall) {
```

Comment:
Can you move this method to right below `updateMap`? This way we keep it consistent with Android Java implementation

Reply:
resolved